scripts/mkcloud

```diff
@@ -1202,6 +1222,11 @@ Optional
         Default value will be applied from Keystone barclamp proposal.
     want_cd='' | 1  (default='')
         Set up a cloud based on the continuous delivery channel.
+    want_ceph=1 (default=1)
```

Comment:
We probably don't need the want_ceph option. I thought setting cephvolumenumber to 0 implies we don't want ceph.

Reply:
Oh maybe, however, if you set cephvolumenumber to 0, then none of the nodes, including the lonelynodes will have volumes for the external ceph cluster. Maybe you can turn it off if your building a cloud, but if you want to build an external ceph cluster you'd need some ceph volumes atleast when you run the lonelynodes step.